libconnman-qt/counter.cpp

```diff
@@ -8,6 +8,7 @@
  */
 
 #include <QtDBus/QDBusConnection>
+#include <QRandomGenerator>
```

Comment:
```
[   60s] counter.cpp:11:10: fatal error: QRandomGenerator: No such file or directory
[   60s]  #include <QRandomGenerator>
[   60s]           ^~~~~~~~~~~~~~~~~~
[   60s] compilation terminated.
[   60s] make[1]: *** [Makefile:607: counter.o] Error 1
```
Qt 5.6

Reply:
Seems QRandomGenerator was added in 5.10

You happy with checking Qt version here and including the header accordingly or you propose a differrent solution?

Reply:
Not being a maintainer of this thing, I guess it's whatever as long as it still compiles against Qt 5.6 (which is currently a requirement for Sailfish OS)

Reply:
I can do a conditional code, depending on old/new Qt version. It should be pretty straightforward.

Reply:
@monich Pushed a fix that should allow it to compile on Qt < 5.10 :)